Trigger/09-Items/Stone Helmet.j

```diff
@@ -0,0 +1,13 @@
+library StoneHelmet requires RandomShit
+
+    function StoneHelmetCast takes unit u returns nothing
+        if UnitHasItemType(u, 'I090') then
```

Comment:
Some basic comments on what these items are or use items from the IdLibrary would be ideal